src/controllers/RoleController.js

```diff
@@ -0,0 +1,34 @@
+import dotenv from 'dotenv';
+import { users, roles } from '../database/models';
+import Customize from '../helpers/Customize';
+
+dotenv.config();
+
+/**
+ * @exports
+ * @class RoleController
+ */
+class RoleController {
+  /**
+   * assign role to users
+   * @static
+   * @param {object} req request object
+   * @param {object} res response object
+   * @memberof RoleController
+   * @returns {object} data
+   */
+  static async assignRole(req, res) {
+    const { id } = req.params;
+    const { email } = req.body;
+
+    await users.update(
+      { roleId: id },
+      { where: { email } }
+    );
+
+    const newRole = await roles.findOne({ where: { id } });
+    return Customize.successMessage(req, res, 'Role assigned successfully', { role: newRole }, 201);
```

Comment:
@NiyoEric maybe you can catch any other error that may arise that you may have not foreseen

Reply:
You are right, thanks to point out.